src/api/docker-files/Dockerfile

```diff
@@ -17,6 +17,14 @@ ADD Gemfile /obs/src/api/Gemfile
 ADD Gemfile.lock /obs/src/api/Gemfile.lock
 RUN chown -R frontend /obs/src/api
 
+# foreman, which we only run in docker, needs a different thor version than OBS.
```

Comment:
Why are you adding this back here? Seems like a mistake.

Reply:
Seems like it, I'll gonna remove it.